website/pages/en/index.js

```diff
@@ -261,13 +261,33 @@ class Index extends React.Component {
         return null
       }
 
-      const showcase = siteConfig.users
-        .filter(user => user.pinned)
-        .map(user => (
-          <a href={user.infoLink} key={user.infoLink}>
-            <img src={user.image} alt={user.caption} title={user.caption} />
-          </a>
-        ))
+      const getRandom = (arr, n) => {
+        var result = new Array(n),
+          len = arr.length,
+          taken = new Array(len)
+        if (n > len) return arr
+        while (n--) {
+          var x = Math.floor(Math.random() * len)
+          result[n] = arr[x in taken ? taken[x] : x]
+          taken[x] = --len in taken ? taken[len] : len
+        }
+        return result
+      }
+
+      const randomizedList = () => {
+        const NUMBER_OF_USERS_TO_SHOW = 3
+
+        return getRandom(
+          siteConfig.users.filter(user => user.pinned),
```

Comment:
Now that we have a randomizer, we should restore the meaning of "pinned" to mean "always on the homepage". Right now we don't want to pin anything, so they can all change to false.

Reply:
Cool, I'll do it and remove the filter.